File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        count=0
        j=0
        for i in range(1, len(nums)):
            if nums[i]!=nums[j]:
                j+=1
                nums[j]=nums[i]
            else:
                count+=1
        for k in range(0, count):
            nums.pop()
            
        return len(nums)

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i=0;
        int j=0;
        while(j<nums.size()){
            if(i==0 || nums[i-1]!=nums[j]){
                nums[i]=nums[j];
                i++;
                j++;
            }else{
                j++;
            }
        }
        return i;
    }
};